packages/target-nodejs-sdk/src/target.js

```diff
@@ -45,6 +47,8 @@ export function executeDelivery(options, decisioningEngine) {
     useBeacon,
     createDeliveryApiMethod = createDeliveryApi
   } = options;
+  const timingTool = createPerfToolInstance();
+  timingTool.timeStart(TIMING_EXECUTE_DELIVERY);
```

Comment:
Let's move the timeStart call to the line before we call `deliveryMethod.execute` so we that we can measure just the time spent inside that function